src/dev-to-git.interface.ts

```diff
@@ -25,6 +25,13 @@ export enum UpdateStatus {
   FAILED_TO_EXTRACT_FRONT_MATTER = 'FailedToExtractFrontMatter',
 }
 
+export interface ConfigurationOptions {
+  silent: boolean;
+  config: string; // the config file path
```

Comment:
Instead of having a comment we could maybe make that more explicit?

Reply:
I left it as `config` to avoid making a breaking change as the library already supports `--config` for passing the options.